punchcut/src/source/glyf.rs

```diff
@@ -57,16 +52,30 @@ mod tests {
         let mut outline = Outline::new();
         for expected_outline in &outlines {
             #[cfg(feature = "hinting")]
-            let mut scaler =
-                Scaler::new(&mut cx, &font, None, expected_outline.size, None, &[]).unwrap();
+            let mut scaler = Scaler::new(
+                &mut cx,
+                &font,
+                None,
+                expected_outline.size,
+                None,
+                &expected_outline.coords,
+            )
+            .unwrap();
```

Comment:
The top level `Context` uses a builder specifically to avoid this sort of confusion. Since this particular code will soon be removed from the public API anyway, I'd like to keep it as is to avoid complicating the higher level code which would have to branch on this rather than passing the fields from the builder directly.

I also haven't added the new `FontId` wrapper type that was previously discussed as a replacement for the raw u64. When that gets done, this could potentially be supplied as `FontId::none()` or `FontId::uncached()` which would remove the ambiguity.